pwndbg/glibc.py

```diff
@@ -17,10 +17,21 @@
 
 safe_lnk = pwndbg.config.Parameter('safe-linking', 'auto', 'whether glibc use safe-linking (on/off/auto)')
 
+glibc_version = pwndbg.config.Parameter('glibc', '', 'GLIBC version for heuristics', scope='heap')
+
 
 @pwndbg.proc.OnlyWhenRunning
-@pwndbg.memoize.reset_on_objfile
 def get_version():
+    if glibc_version.value:
+        ret = re.search(r"(\d+)\.(\d+)", glibc_version.value)
+        if ret:
+            return tuple(int(_) for _ in ret.groups())
```

Comment:
If user sets this incorrectly, we should either show him immediately that it's incorrect OR show here that the set version cannot be parsed and should be set in a given format.

Reply:
Ok, I'll add this feature!